web/src/compose_validate.ts

```diff
@@ -500,28 +536,28 @@ function wildcard_mention_policy_authorizes_user() {
             return true;
         }
         const person = people.get_by_user_id(current_user.user_id);
-        const current_datetime = new Date(Date.now());
-        const person_date_joined = new Date(person.date_joined);
+        const current_datetime = new Date(Date.now()).getTime();
+        const person_date_joined = new Date(person.date_joined).getTime();
```

Comment:
This seems like a functional change?

Reply:
Yes. Typescript is unable to perform arithmetic operations on the `Date` objects directly as seen below. It can only do that on primitive types. The `getTime` method returns the `Date` objects as `number`.

```
const days = (current_datetime - person_date_joined) / 1000 / 86400;
```